core/runtime/src/main/java/io/quarkus/runtime/graal/BootLoaderSubstitutions.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.runtime.graal;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+import com.oracle.svm.core.jdk.JDK11OrLater;
+
+@TargetClass(className = "jdk.internal.loader.BootLoader", onlyWith = JDK11OrLater.class)
+final class Target_jdk_internal_loader_BootLoader {
```

Comment:
please add a comment explaining why we added that one with a pointer to the GraalVM issue.

Reply:
I was hoping the GraalVM issue would be labeled as a bug yesterday and waited for that to include such a comment, but it's still not recognized as a bug. I'll add a comment about the issue anyway.

Reply:
Done.